airbyte-webapp/STYLE-GUIDE.md

```diff
@@ -0,0 +1,56 @@
+# Frontend Style Guide
+
+This serves as a living document regarding conventions we have agreed upon as a frontend team. In general, the aim of these decisions and discussions is to both (a) increase the readability and consistency of our code and (b) decrease day to day decision-making so we can spend more time writing better code.  
+
+## General Code Style and Formatting
+
+* Where possible, we rely on automated systems to maintain consistency in code style 
+* We use eslint, Prettier, and VSCode settings to automate these choices.  The configuration files for these are checked into our repository, so no individual setup should be required beyond ensuring the [eslint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) and [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) extensions are installed in VSCode
```

Comment:
The prettier extension should not be required (I don't have it installed), since we run prettier via ESLint, and it can autofix via it, it should be enough to have the eslint extension installed and the following in your configuration:

```js
"editor.codeActionsOnSave": {
  "source.fixAll.eslint": true,
},
```